lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -135,6 +145,19 @@ def auth_ldap(self, **kwargs):
 
         self.client.auth_ldap(username, password, mount_point)
 
+    def boolean_or_cacert(self, arg):
```

Comment:
I think a `validate_certs`  + optional `cacert` param would be good. I agree on the SSL/TLS verify stuff, and I have no issues with 'new' args but with args that are commonly labeled across other Ansible _things_ (like `state`, validate_certs`, etc)  it's preferred for it to behave in similar way across the board.